tests/test_requests.py

```diff
@@ -1042,6 +1042,22 @@ def test_response_decode_unicode(self):
         chunks = r.iter_content(decode_unicode=True)
         assert all(isinstance(chunk, str) for chunk in chunks)
 
+        # raise an exception if encoding isn't set
+        r = requests.Response()
+        r.raw = io.BytesIO(b'the content')
+        r.encoding = None
+
+        with pytest.raises(TypeError):
+            chunks = r.iter_content(decode_unicode=True)
+
+        # raises an exception if the encoding is garbage
+        r = requests.Response()
+        r.raw = io.BytesIO(b'the content')
+        r.encoding = 'invalid encoding'
+
+        with pytest.raises(LookupError):
+            chunks = r.iter_content(decode_unicode=True)
+
```

Comment:
No, I think I just added them to other without really thinking about it. I've parametrized and committed this. Need to figure out why it's not showing up.